### PR TITLE
BrainBot repeater

### PR DESCRIPTION
**Pull request by LukeG294**
Pretty simple, but I think a lot of users would enjoy this. The cool down is shared with the "someone tell me to" auto response as they are somewhat similar.

### CODE REVIEW

**Comment by LukeG294** [comment]
When we've been testing this, I notice that a lot of people try to configure their wording with the "someone tell me to" command, and I think that it would be nice just to have a repeat function.

**Comment by LukeG294** [comment]
Note: If this is accepted the wiki commands should be updated @LukeG294 @bsoyka

File: main.py [hunk 1]
```diff
@@ -103,6 +103,15 @@ async def on_chat(msg):
                         await send_message(f"@{user.get_username()}: {to_do}", bot_chat)
                     else:
                         console.log("Cancelled due to cooldown")
+                
+                elif msg.text.lower().startswith("!repeat"):
+                    msg_text = msg.text[19:]
```
Comment (bsoyka) [inline main.py:108]:
```suggestion
                    msg_text = msg.text[8:]
```

File: main.py [hunk 2]
```diff
@@ -103,6 +103,15 @@ async def on_chat(msg):
...
+                    console.log(f"Repeating {user.get_username()}")
+                    if tell_me_to_cooldown.run():
+                        await send_message(f"{msg_text}", bot_chat)
```
Comment (bsoyka) [inline main.py:111]: I think the user that used the command should be mentioned somewhere, especially for Ryver organizations that allow users to remove their own messages.

**Review by bsoyka — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (LukeG294) [reply main.py:111]: 1: I definitely agree with that, we wouldn't want anyone using profanity and then making it look like BrainBot just did it out of nowhere. I'll try and add something for that.

2: Quick question, what does msg.text[19:] mean?

Comment (bsoyka) [reply main.py:111]: It takes the message text and gets everything after the 19th character.
```py
>>> text = "abcde"
>>> text[3:]
'de'
>>> text[:3]
'abc'
```

**Review by bsoyka — APPROVED** [review]

**Pull request merged**